common/djangoapps/mailchimp_pipeline/tests/helpers.py

```diff
@@ -1,18 +1,20 @@
 """
-    Helper functions requried for tests
+Helper functions required for tests
 """
 import hashlib
-from django.conf import settings
 from datetime import datetime, timedelta
+
+from django.conf import settings
+
 from lms.djangoapps.onboarding.models import OrganizationPartner
 from lms.djangoapps.onboarding.tests.factories import OrganizationFactory
 
 
 def create_organization(user):
     """
-        create and return organziation with given user as an organization admin
-        :param user: organization admin user
-        :return: created organziation
+    create and return organization with given user as an organization admin
+    :param user: organization admin user
+    :return: created organization
```

Comment:
Lets use the same format for docstring here as being used everywhere else
```
"""
Create and return organisation with given user as an organisation admin

Arguments:
    User: organisation admin user

Returns:
    Created organisation
"""
```

Reply:
Not under the scope of the ticket. These docstrings were already added, I just fixed the spelling mistake in some of these.

Reply:
Any docstring newly added is in this format.